### PR TITLE
Upgrade `PkgLicenses` to be 1.0-compatible

### PR DESCRIPTION
**Pull request by staticfloat**

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1 +1,2 @@
 julia 0.6
+Test
```
Comment (Keno) [inline REQUIRE]: stdlibs don't go into REQUIRE

File: Manifest.toml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[[Base64]]
```
Comment (Keno) [inline Manifest.toml:1]: I don't think we need to check in the Manifest. Libraries usually don't.

File: Project.toml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+name = "PkgLicenses"
+uuid = "fc669557-7ec9-5e45-bca9-462afbc28879"
```
Comment (Keno) [inline Project.toml:2]: I assume you've made sure this matches what's in the registry?

File: REQUIRE [hunk 4]
```diff
@@ -1 +1,2 @@
 julia 0.6
```
Comment (Keno) [inline REQUIRE]: Need to drop 0.6 here since you're using 0.7-only stuff below.

File: Project.toml (hunk 3 above)
Comment (staticfloat) [reply Project.toml:2]: Yes.  I copied it form there.

File: Manifest.toml (hunk 2 above)
Comment (staticfloat) [reply Manifest.toml:1]: I had a long discussion about this with Stefan.  You should read the thread in #pkg-usage.  I'll link you on Slack.

**Comment by Keno** [comment]
Femtocleaner has some comments.

**Comment by staticfloat** [comment]
Cool.  BTW I found the FemtoCleaner output kind of confusing; I couldn't really tell what it was going to change things to, and I just blindly clicked "Fix It" after not being able to tell what the changes it _would_ apply would be.

File: src/PkgLicenses.jl [hunk 5]
```diff
@@ -40,10 +40,10 @@ function license(io::IO, lic::AbstractString="")
         try
             println(io, readlicense(lic))
         catch
-            print_with_color(io, :red, "License $lic is not available.")
+            printstyled(:red, "License $lic is not available.", color=io)
```
Comment (Keno) [inline src/PkgLicenses.jl]: oops

Comment (Keno) [reply src/PkgLicenses.jl]: (I think this was wrong before and the transform is technically correct).

Comment (staticfloat) [reply src/PkgLicenses.jl]: Yeah, that does look wrong.  ;)

Comment (staticfloat) [reply src/PkgLicenses.jl]: In BB, we just catch any errors, so it wouldn't be noticed there.

**Review by Keno — APPROVED** [review]

**Pull request merged**